src/ansible/roles/dns/templates/etc.dnsmasq.conf.j2

```diff
@@ -0,0 +1,81 @@
+# This config is specific to our use and does not reflect multiple AD DCs
```

Comment:
This config is specific to downstream CI use case

Reply:
Rephrased.